### ISSUE TITLE
DSL multi-line indentation

### ISSUE THREAD

**Issue by spydon** [scalastyle/scalastyle#226]
Is it possible to make the indentation checker follow this http://docs.scala-lang.org/style/declarations.html#multiple-parameter-lists ?

So that it doesn't complain on things like:
```
protected def forResource(resourceInfo: Any)
                         (f: (JsonNode) => Any) 
                         (implicit urlCreator: URLCreator, configurer: OAuthConfiguration): Any = {
      ...
}
```

**Comment by matthewfarwell** [comment]
This is possible. You would have to write a checker which makes sure that if you have multiple parameter lists, each one is on a different line. If you want to do this, please go ahead and submit a PR.
